media/js/freesound.js

```diff
@@ -126,3 +126,64 @@ function show_hide_bookmark_form(id)
     }
     $("#bookmark_form_"  + id.toString()).toggle();
 }
+
+// FOOTER BANNER
+function hideFooterBanner(){
+    $('#fotterBanner').hide(200);
+}
+
+function showFooterBanner() {
+    $('#fotterBanner').show(200);
+}
+
+// Modals
+function hideModal(){
+    $("#fsmodal").hide();
+}
+
+function openModal(){
+    $("#fsmodal").show();
+}
+
+function generateModalHTML(title, contents){
+    return '<div id="fsmodal" class="modal"><div class="modal-content"><div class="modal-header"><span onclick="hideModal();" class="close">&times;</span>'
```

Comment:
The idea was to have it as generic as possible and easy to reuse, like you can just call this function anywhere and it will generate the modal...